.travis.yml

```diff
@@ -50,9 +50,8 @@ install:
   - sudo make install
 script:
   - echo "build successful"
-  - echo "cd ~/ngxpagespeed"
-  - echo "sudo ./test/run_tests.sh 8050 8051 $HOME/mod_pagespeed/src $HOME/nginx-branches-default/objs selfsigned.modpagespeed.com"
-  - echo "sudo ./test/run_tests.sh 8050 8051 $HOME/mod_pagespeed $HOME/nginx-branches-default/objs/nginx selfsigned.modpagespeed.com"
+  - cd ~/ngxpagespeed
+  - sudo ./test/run_tests.sh $HOME/mod_pagespeed $HOME/nginx-branches-default/objs/nginx
```

Comment:
Every PR already gets a travis failure right now about tools/clang/scripts/update.py

In general, if a PR successfully builds with travis it should pss tests, so enabling this seems right to me?